### PR TITLE
Varvara/feature/google auth for ios

### PR DESCRIPTION
**Pull request by varvarashcherbyna7**

### CODE REVIEW

File: src/screens/Main.tsx [hunk 1]
```diff
@@ -39,33 +39,33 @@ interface IMain {
 }
 
 const Main: React.FC<IMain> = ({navigation, route}) => {
-  const [isCurrentUser, setIsCurrentUser] = useState<boolean>(false);
+  // const [isCurrentUser, setIsCurrentUser] = useState<boolean>(false);
```
Comment (kipod) [inline src/screens/Main.tsx:42]: Pls remove

File: src/screens/Main.tsx [hunk 2]
```diff
@@ -39,33 +39,33 @@ interface IMain {
...
     setActiveTab(routName ?? OffersScreen);
   }, [routName]);
 
-  useQuery(
+  const {data} = useQuery(
     [WHOAMI],
     async () => {
       const res = await WhoamiService.whoamiApiWhoamiUserGet();
-      setIsCurrentUser(res);
+      // setIsCurrentUser(res);
```
Comment (kipod) [inline src/screens/Main.tsx:54]: Pls remove

**Review by kipod — APPROVED** [review]
Thank you

File: src/screens/LetGetStart.tsx [hunk 3]
```diff
@@ -8,32 +9,87 @@ import {
...
+import {useFilteringData} from '../store/filteringData';
+import {MultiSelect} from 'react-native-element-dropdown';
+import colors from '../assets/colors';
 
 interface ILetGetStart {
   navigation?: any;
 }
 
 const LetGetStart: React.FC<ILetGetStart> = ({navigation}) => {
+  const [locationsData, setLocationsData] = useState<Location[]>([]);
```
Comment (Taraslysak) [inline src/screens/LetGetStart.tsx:25]: redundant state. Refactoring required

**Review by Taraslysak — CHANGES_REQUESTED** [review]
Please, do some refactoring

**Pull request merged**